### ISSUE TITLE
Extra characters in Space names after upgrade to v0.15.0

### ISSUE THREAD

**Issue by philrz** [brimsec/zq#1111]
Discovered while doing a smoke test on the Brim release candidate tagged `v0.15.0`.

I had Spaces that had been created when running `v0.14.0` named "hello.pcapng.brim" and "wrccdc.pcap.brim", created by importing pcaps. After the upgrade, the Space names visible in the app had extra characters visible in them.

![image](https://user-images.githubusercontent.com/5934157/90303039-62dafe80-de5f-11ea-8fcc-68542d59156f.png)

**Comment by philrz** [comment]
@jameskerr: Yeah, @mattnibs indicated that a PR of his might have been the culprit, so I've transferred this issue to the `zq` repo. I guess I'll assign it to him for now while he determines if his theory is correct.

**Comment by philrz** [comment]
Verified in Brim commit `ae2b811` ... talking to `zqd` commit `8a770dd`.

Just to revisit how to repro the problem in Dev mode, after clearing all app state & Space directories, I do:

```
git checkout v0.14.0
npm install
npm start
```

That starts a `zqd` tagged `v0.18.0` as well. At this point I can import a small pcap and observe the Space created with the expected name:

![image](https://user-images.githubusercontent.com/5934157/90434710-bbd6ac80-e082-11ea-8fec-762cb19bc8a4.png)

I quit the app and now do:

```
git checkout v0.15.0
npm install
npm start
```

That starts a `zqd` tagged `v0.19.0` which has the problem captured in this issue. The problem shows up in the app with the mangled Space name:

![image](https://user-images.githubusercontent.com/5934157/90435005-343d6d80-e083-11ea-9422-47eeebb29c03.png)

To verify the fix, the first repro steps at `v0.14.0` are the same. Now after quitting the app I do:

```
git checkout ae2b811
npm install
npm start
```

That starts a `zqd` tagged `8a770dd` that includes the fix. The app no longer shows the mangled Space name.

![image](https://user-images.githubusercontent.com/5934157/90435446-e07f5400-e083-11ea-857e-49a34f3e197d.png)

Thanks @mattnibs!
